### DISCUSSION TITLE
composite search with objectType: "Question" is not working

### DISCUSSION THREAD

**Discussion by bharathravi-in** [Sunbird-Ed/Community#387] [General Discussion]
I am facing issue in Question set when i click on ADD FROM LIBRARY the bellow one is the request to get objectType:["Question"].
but i am getting count 0 only.

=> i have created object category of Multiple choice Question and objectType as Question.

curl --location --request POST 'https://uphrh.in/api/composite/v1/search' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer <token>' \
--data-raw '{
    "request": {
        "query": "",
        "filters": {
            "primaryCategory": [
                "Multiple Choice Question"
            ],
            "status": [
                "Live"
            ],
            "objectType": [
                "Question"
            ]
        },
        "sort_by": {
            "lastUpdatedOn": "desc"
        }
    }
}'

I need list of questions to be getting in the response. please guide me to get it.
@vinukumar-vs @Krishnaj20

**Comment by Krishnaj20** [comment]
@pallakartheekreddy - can you check and respond please?

**Comment by pallakartheekreddy** [comment]
@BharathTarento I guess you created MCQ Question under Question Set right? If yes, those questions will not appear in search as those questions created with Visibility as `Parent` means those questions will be to exist only under that Question Set. 

@Krishnaj20 @surendrasinghs

**Reply by bharathravi-in** [reply]
thank you @pallakartheekreddy it worked for me. 
Could you please elaborate more for Visibility **Private and Protected** Functionalities in the system.

**Reply by Krishnaj20** [reply]
@maheshkumargangula - can you clarify if 'private' & 'protected' are implemented in Knowlg BB?
